pkg/executables/kubectl.go

```diff
@@ -393,6 +410,15 @@ func (k *Kubectl) ValidateClustersCRD(ctx context.Context, cluster *types.Cluste
 	return nil
 }
 
+func (k *Kubectl) ValidateEKSAClustersCRD(ctx context.Context, cluster *types.Cluster) error {
```

Comment:
maybe these methods should be called something like `EksaClusterCrdExists` (or for the above `ValidateClustersCRD` could be `CapiClusterCrdExists`)? `Validate` seems a bit overloaded.